src/features/embalm/stepContent/hooks/useCreateSarcophagus/useCreateSarcophagus.ts

```diff
@@ -23,16 +23,33 @@ export function useCreateSarcophagus(
   const [currentStage, setCurrentStage] = useState(CreateSarcophagusStage.NOT_STARTED);
   const [stageExecuting, setStageExecuting] = useState(false);
   const [stageError, setStageError] = useState<string>();
+  const [stageInfo, setStageInfo] = useState<string>();
   const [isStageRetry, setIsStageRetry] = useState(false);
 
   // Each hook represents a stage in the create sarcophagus process
   const { dialSelectedArchaeologists } = useDialArchaeologists();
   const { initiateSarcophagusNegotiation } = useArchaeologistSignatureNegotiation();
-  const { uploadAndSetArweavePayload } = useUploadFileAndKeyShares();
+  const { uploadAndSetArweavePayload, uploadStep } = useUploadFileAndKeyShares();
   const { approveSarcoToken } = useApproveSarcoToken(sarcoToken);
   const { submitSarcophagus } = useSubmitSarcophagus(embalmerFacet);
   const { clearSarcophagusState, successData } = useClearSarcophagusState();
 
+  const stageInfoMap = useMemo(() => {
+    return new Map<CreateSarcophagusStage, string>([
+      [CreateSarcophagusStage.DIAL_ARCHAEOLOGISTS, ''],
+      [CreateSarcophagusStage.ARCHAEOLOGIST_NEGOTIATION, ''],
+      [CreateSarcophagusStage.UPLOAD_PAYLOAD, uploadStep],
+      [CreateSarcophagusStage.APPROVE, ''],
+      [CreateSarcophagusStage.SUBMIT_SARCOPHAGUS, 'Waiting for transaction'],
+      [CreateSarcophagusStage.CLEAR_STATE, ''],
+      [CreateSarcophagusStage.COMPLETED, ''],
+    ]);
+  }, [uploadStep]);
```

Comment:
There's probably a better way of doing this, but this map is meant to be used to show extra UI information during a stage that takes a little longer than normal. Used similarly to the stage->stageFunction mapping